metaflow/plugins/cards/card_modules/convert_to_native_type.py

```diff
@@ -348,11 +348,11 @@ def _match_partial_type():
         try:
             col_type = str(column_object.dtype)
             if col_type in type_parser:
-                return type_parser[col_type](column_object)
+                return type_parser[col_type](column_object.fillna("null"))
             else:
                 parsed_col = _match_partial_type()
                 if parsed_col is not None:
-                    return parsed_col
+                    return parsed_col.fillna("null")
             return truncate_long_objects(column_object)
```

Comment:
Should we also add `fill_null` to the column object going to the `truncate_long_objects` function.

Reply:
the main reason for asking for this change is because the last `return` statement is only encountered when there is a type that we couldn't be parsed. Ie. it doesn't belong to either the `type_parsers` or the `partial_type_name_match_parsers`;

Reply:
Hmm, well `truncate_long_objects` does already convert everything to strings, so it shouldn't be necessary to prevent the parse errors in the browser, but it would make nulls appear more consistently between column types.